neuromation/api/parsing_utils.py

```diff
@@ -158,14 +165,25 @@ def _split_image_name(
     ) -> Tuple[str, Optional[str]]:
         colon_count = image.count(":")
         if colon_count == 0:
-            image, tag = image, default_tag
+            name, tag = image, default_tag
         elif colon_count == 1:
-            image, tag = image.split(":")
-            if not tag:
+            # case `ubuntu:latest`
+            name, tag = image.split(":")
+            if "/" in tag:
+                # case `localhost:5000/ubuntu`
+                name, tag = image, default_tag
+            elif not tag:
                 raise ValueError("empty tag is not allowed")
+        elif colon_count == 2:
+            # case `localhost:9000/owner/ubuntu:latest`
+            if "/" not in image:
+                # case `ubuntu:v11:latest`
+                raise ValueError("too many tags")
+            *image_arr, tag = image.split(":")
+            name = ":".join(image_arr)
```

Comment:
It could be written shorter as `name, tag = image.rsplit(':', 1)`.

But what about "localhost/owner/ubuntu:v11:latest"? It is passed in contrary to "ubuntu:v11:latest".

Can a tag contain a slash? It is forbidden in some cases ("ubuntu:v18/04"), but is allowed in others ("localhost:9000/owner/ubuntu:v18/04").

Also the case of an empty tag is not handled here ("localhost:9000/owner/ubuntu:").

Reply:
> It could be written shorter as name, tag = image.rsplit(':', 1).

ok thanks!

> Can a tag contain a slash? It is forbidden in some cases ("ubuntu:v18/04"), but is allowed in others ("localhost:9000/owner/ubuntu:v18/04").

Indeed, docker forbids both cases:
```
$ docker pull ubuntu:10/04
Using default tag: latest
Error response from daemon: Get https://ubuntu:10/v2/: dial tcp: lookup ubuntu: no such host
$ docker pull localhost:5000/ubuntu:10/04
invalid reference format
```
added a check.

> Also the case of an empty tag is not handled here ("localhost:9000/owner/ubuntu:").

supported, thanks